src/NServiceBus.Transport.Msmq/CheckMachineNameForCompliance.cs

```diff
@@ -1,18 +1,18 @@
 namespace NServiceBus.Transport.Msmq
 {
+    using System;
     using System.Runtime.InteropServices;
     using System.Text;
-    using Logging;
 
-    class CheckMachineNameForComplianceWithDtcLimitation
+    static class CheckMachineNameForCompliance
     {
         [DllImport("kernel32.dll", SetLastError = true, CharSet = CharSet.Auto)]
         static extern bool GetComputerNameEx(COMPUTER_NAME_FORMAT nameType, [Out] StringBuilder lpBuffer, ref uint lpnSize);
 
         /// <summary>
         /// Method invoked to run custom code.
         /// </summary>
-        public void Check()
```

Comment:
pretty sure the above doco comment adds no value ?

Reply:
@SimonCropp - yeah, I agree. Although I didn't modify it, it makes sense to change it as part of this PR.